src/api.jl

```diff
@@ -648,6 +648,11 @@ function get_transaction_results(ctx::Context, id::AbstractString; kw...)
     return _parse_multipart_results_response(rsp)
 end
 
+function cancel_transaction(ctx::Context, id::AbstractString; kw...)
+    path = PATH_ASYNC_TRANSACTIONS * "/$id/cancel"
+    return _post(ctx, path)
+end
```

Comment:
Can you please add a unit test for this? Probably it doesn't need to be an integration test; just a unit test that confirms that the correct path is hit with a POST request?

Thank you!